Server 1978 install doc updates (#7156)
circleci/circleci-docs: jekyll/_cci2/server/installation/phase-2-core-services.adoc

[#pull-image-from-dockerhub]
== 2. Pull image from DockerHub

> Credentials to pull the images from CircleCI's image registry will be provided to you as part of the onboarding process. A `docker-registry` Kubernetes secret will be used to pull images from ACR. You have two options, depending on whether your application has access to the public internet.

Reviewer: ```suggestion
Credentials to pull the images from CircleCI's image registry will be provided to you as part of the onboarding process. A `docker-registry` Kubernetes secret will be used to pull images from Azure Container Registry (ACR). You have two options, depending on whether your application has access to the public internet.
```

> CircleCI's images are stored in Azure Container Registry(ACR). Credentials to pull the images from CircleCI's ACR will be provided to you as part of the onboarding process. A `docker-registry` Kubernetes secret will be used to pull images from ACR. You have two options, depending on whether you application has access to the public internet.

Reviewer 2: ```suggestion
Credentials to pull the images from CircleCI's image registry will be provided to you as part of the onboarding process. A `docker-registry` Kubernetes secret will be used to pull images from ACR. You have two options, depending on whether your application has access to the public internet.
```

[.tab.dockerhub.Public]
--
**Option 1:** Your application has access to the public internet.

> This example creates a secret which enables deployments to pull images from CircleCI's image registry. The `docker-registry` kubernetes secret will take the following form:

Reviewer: ```suggestion
This example creates a secret to enable deployments to pull images from CircleCI's image registry. The `docker-registry` Kubernetes secret takes the following form:
```

> This example creates a secret which enables deployments to pull images from CircleCI's ACR registry. The `docker-registry` kubernetes secret will take the following form:

Reviewer 2: ```suggestion
This example creates a secret which enables deployments to pull images from CircleCI's image registry. The `docker-registry` kubernetes secret will take the following form:
```

[source,shell]
----
kubectl create secret docker-registry regcred \
  --docker-server=cciserver.azurecr.io \
  --docker-username=<your-username> \
  --docker-password="<provided-token>" \
  --docker-email=<your-contact-email>
----
--

[.tab.dockerhub.Private]
--
**Option 2:** Your application does NOT have access to the public internet.

> The credentials provided to you will allow you to pull and store copies of our images locally. Pull and store the images in whatever docker repository you have available. The `docker-registry` kubernetes secret will take the following form:

Reviewer: ```suggestion
The credentials provided to you allow you to pull and store copies of our images locally. Pull and store the images in whichever docker repository you have available. The `docker-registry` Kubernetes secret takes the following form:
```

  license: <license>
----

[#Registry]
==== Registry

> The registry to pull images from will have been provided by Circle, or you may have added the images to your own hosted registry. You will need to add it to `values.yaml`:

Reviewer: ```suggestion
The registry to pull images from will have been provided to you, or you may have added the images to your own hosted registry. You will need to add the registry to `values.yaml`:
```
